Yoast/wordpress-seo
# Lingo 972 add assessors for blog pages and posts

## Context


*

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Adds assessors for blog posts and pages, and set them in `analysis.worker` in content analysis app.

## Relevant technical choices:

*

## Test instructions

### Test instructions for the acceptance test before the PR gets merged
This PR can be acceptance tested by following these steps:

- Make the following changes in the code:

In `apps/content-analysis/src/App.js` on the line before `worker.initialize( config )`, please add this (don't commit):

```
		config.customAnalysisType = "storePostsAndPages";
```


- Execute the following steps in the content analysis app.
- Make sure that the toggle `keyphrase distribution` ​is switched off for all of the tests in the content analysis app.

#### Test readability/SEO (regular)
- Make sure that the toggle `Is a cornerstone article` is switched off.
- Make sure that the toggle `Is a related keyphrase` ​is switched off.
- Set the analysis language to `en_US`.
- Add a text with 200 words and 1 image.
- In the SEO analysis, confirm that you don't see the following assessments:
  - Images and Videos
  - Image alt tags
- In the readability analysis, confirm that you don't see the following assessments:
  - Lists 
- Confirm that the assessor _is not_ a cornerstone assessor:
  - Readability: add a `<h2>subheading</h2>` followed by a text of 320 words. The bullet for the `Subheading distribution` assessment should be orange.
  - SEO: remove all text. The `Text length` assessment should recommended minimum of 300 words.
  
#### Test readability/SEO (cornerstone)
- Switch on the toggle `Is a cornerstone article`
- In the readability analysis, confirm that you don't see the following assessments:
  - Lists
- In the SEO analysis, confirm that you don't see the following assessments:
  - Images and Videos
  - Image alt tags
- Confirm that the assessor _is_ a cornerstone assessor:
  - Readability: add a `<h2>subheading</h2>` followed by a text of 320 words. The bullet for the `Subheading distribution` assessment should be red.
   - SEO: remove all text. The `Text length` assessment should recommended minimum of 900 words.

#### Test related keyphrase analysis (regular)
- Switch **on** the toggle `Is a related keyphrase`
- Switch **off** the toggle `Is a cornerstone article`
- Set the keyphrase to `vegan recipes`
- Add a text two images with alt tags that don't contain keyphrase.
```
<img src="img_cat.jpg" alt="beautiful tortie cat" width="500" height="600">
<img src="img_girl.jpg" alt="a girl sitting on a chair" width="500" height="600">
```
- Confirm that `Image keyphrase` assessment turns orange.

#### Test related keyphrase analysis (cornerstone)
- Switch **on** the toggle `Is a related keyphrase`
- Switch **on** the toggle `Is a cornerstone article`
- Set the keyphrase to `vegan recipes`
- Add a text two images with alt tags that don't contain keyphrase.
```
<img src="img_cat.jpg" alt="beautiful tortie cat" width="500" height="600">
<img src="img_girl.jpg" alt="a girl sitting on a chair" width="500" height="600">
```
- Confirm that `Image keyphrase` assessment turns red.



### Test instructions for QA when the code is in the RC




QA can test this PR by following these steps:

*

## Impact check

This PR affects the following parts of the plugin, which may require extra testing:

*

## UI changes


## Documentation


## Quality assurance


Fixes https://yoast.atlassian.net/browse/LINGO-972
